test/src/kbasesearchengine/test/main/NarrativeInfoDecoratorTest.java

```diff
@@ -9,12 +9,14 @@
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
+import java.net.URL;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import kbasesearchengine.authorization.TemporaryAuth2Client;
```

Comment:
import not used. 

Also, this one is not used too 

import static kbasesearchengine.test.events.handler.WorkspaceEventHandlerTest.wsTuple;

Reply:
Done